src/exodus/tx.cpp

```diff
@@ -797,6 +800,26 @@ bool CMPTransaction::interpret_UnfreezeTokens()
     return true;
 }
 
+/** Tx 1025 */
+bool CMPTransaction::interpret_CreateDenomination()
+{
+    if (pkt_size < 16) {
+        return false;
+    }
+
+    memcpy(&property, &pkt[4], 4);
```

Comment:
better to use `sizeof(property)` instead.

Reply:
If we use this way we need to use it for `pkt[4]` too, which will loo ugly for the late member (e.g. `memcpy(&nValue, &pkt[sizeof(type) + sizeof(version) + sizeof(property)], sizeof(nValue));`). My idea is we should keep this way for now and find a better way and refactor it later.